test-specifications/crypto-service/accountCreateTransaction.md

```diff
@@ -0,0 +1,439 @@
+    "maxAutoTokenAssociations": 100
+  }
+}
+```
+
+#### JSON Response Example
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 99232,
+  "result": {
+    "accountId": "0.0.12345",
+    "status": "SUCCESS"
+  }
+}
+```
+
+### **Staked ID:**
+
+- ID of the account to which this account is staked.
+  - OR
+- ID of the node to which this account is staked.
+
+| Test no | Name                                                                                  | Input                                                                                        | Expected response                                                                                     | Implemented (Y/N) |
+|---------|---------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------|-------------------|
+| 1       | Creates an account with the staked account ID set to the operators account ID         | key=<VALID_KEY>, stakedAccountId=<OPERATOR_ACCOUNT_ID>                                       | The account creation succeeds and the account has a staking account ID equal to the input account ID. | N                 |
+| 2       | Creates an account with the staked node ID set to a valid node ID                     | key=<VALID_KEY>, stakedNodeId=<VALID_NETWORK_NODE_ID>                                        | The account creation succeeds and the account has a staking node ID equal to the input node ID.       | N                 |
+| 3       | Creates an account with the staked account ID set to an account ID that doesn't exist | key=<VALID_KEY>, stakedAccountId="123.456.789"                                               | The account creation fails with an INVALID_STAKING_ID response code from the network.                 | N                 |
+| 4       | Creates an account with the staked node ID set to a node ID that doesn't exist        | key=<VALID_KEY>, stakedNodeId=123456789                                                      | The account creation fails with an INVALID_STAKING_ID response code from the network.                 | N                 |
+| 5       | Creates an account with the staked account ID set to an empty account ID              | key=<VALID_KEY>, stakedAccountId=""                                                          | The account creation fails with and SDK internal error.                                               | N                 |
+| 6       | Creates an account with the staked account ID set to an invalid node ID               | key=<VALID_KEY>, stakedNodeId=-100                                                           | The account creation fails with an INVALID_STAKING_ID response code from the network.                 | N                 |
```

Comment:
I think its about an account with the staked node ID (not an account ID) set to an invalid node ID.

Also maybe should be similar test for an account with the staked account ID set to an invalid node ID.

Reply:
Yeah this was a typo, I'll fix it.

Reply:
Fixed